pkg/controller/masteruserrecord/sync_test.go

```diff
@@ -159,24 +149,45 @@ func TestSynchronizeSpec(t *testing.T) {
 
 func TestSynchronizeStatus(t *testing.T) {
 	// given
-	logf.SetLogger(logf.ZapLogger(true))
```

Comment:
oh, why did you remove these lines? I find it useful to have the `INFO` log messages.
(It's not a requirement to revert the change, though)

Reply:
because I moved it to `prepareSynchronizer` function - it was there before, so it was just a duplication:
https://github.com/codeready-toolchain/host-operator/blob/5361b85e3114bbb7972a88536f0486c159b7c9b4/pkg/controller/masteruserrecord/sync_test.go#L641-L649